apps/ledger-live-mobile/src/components/Web3AppWebview/PlatformAPIWebview.tsx

```diff
@@ -46,8 +46,20 @@ import { RootNavigationComposite, StackNavigatorNavigation } from "../RootNaviga
 import { BaseNavigatorStackParamList } from "../RootNavigator/types/BaseNavigator";
 import { WebviewAPI, WebviewProps } from "./types";
 import { useWebviewState } from "./helpers";
-
-const tracking = trackingWrapper(track);
+import { currentRouteNameRef } from "../../analytics/screenRefs";
+
+const tracking = (isNativeSwap?: boolean) =>
```

Comment:
```suggestion
const trackWithNativeFlowOrigin = (isNativeSwap?: boolean) =>
```

Reply:
native swap is not in a webview.

Reply:
In that case it's just `trackWithDiscoverOrigin`

Reply:
Ok I am looking at the Slack thread now